src/lib/components/container.svelte

```diff
@@ -1,24 +1,38 @@
 <script lang="ts">
 	export let direction: 'row' | 'column' = 'column'
 	export let gap = 0
+	export let variant: 'space-between' | 'flex-start' | 'flex-end' | 'center' = 'flex-start'
```

Comment:
I think we could name this `justify` instead of variant.

Reply:
Agreed